src/components/FilmStills/FilmStillsForm.tsx

```diff
@@ -0,0 +1,105 @@
+import { useState } from 'react';
+import { postData } from '../../utils/apiCalls';
+import {
+  getPresignedUrl,
+  putToAWS,
+  postStillOrTeamToDB,
+} from '../../awsS3/helperFunctions';
+import FormControl from '@mui/material/FormControl';
+import TextField from '@mui/material/TextField';
+import Button from '@mui/material/Button';
+
+interface IFilmStillsForm {
+  setIsEditing: any;
+  isEditing: boolean;
+  epk_id: any;
+  filmStills: any;
+  setFilmStills: any;
+}
+
+const FilmStillsForm = ({
+  epk_id,
+  setIsEditing,
+  filmStills,
+  setFilmStills,
+  isEditing,
+}: IFilmStillsForm) => {
+  const [description, setDescription] = useState<string>('');
+
+  const postFilmStill = (filmStill: object) => {
+    postData('https://epk-be.herokuapp.com/api/v1/film_stills', filmStill)
+      .then((data: any) => {
+        handleImageSubmit(data.data.id).then((data) =>
+          setFilmStills([...filmStills, data.data])
+        );
+      })
+      .catch((err) => console.log(err));
```

Comment:
We probably want to add an actual error action here, instead of just the console.log. Just noting this for later